.gitignore

```diff
@@ -6,3 +6,4 @@ tags
 Makefile
 node_modules/
 examples/node/svgdump/
+examples/node/pdf2png/helloworld.png
```

Comment:
Nit: it might be better to use `examples/node/pdf2png/*.png` instead, to simply ignore all PNG images regardless of name.